src/managers/KeybindManager.cpp

```diff
@@ -142,7 +142,7 @@ void CKeybindManager::updateXKBTranslationState() {
     xkb_rule_names rules = {.rules = RULES.c_str(), .model = MODEL.c_str(), .layout = LAYOUT.c_str(), .variant = VARIANT.c_str(), .options = OPTIONS.c_str()};
 
     const auto     PCONTEXT   = xkb_context_new(XKB_CONTEXT_NO_FLAGS);
-    FILE* const    KEYMAPFILE = FILEPATH == "" ? NULL : fopen(FILEPATH.c_str(), "r");
+    FILE* const    KEYMAPFILE = FILEPATH == "" ? NULL : fopen(absolutePath(FILEPATH, g_pConfigManager->configCurrentPath).c_str(), "r");
```

Comment:
This change doesn't quite fix the issue, right? Now `FILEPATH` is correctly expanded, but if `FILEPATH` is non-empty but refers to a nonexistent file, this will still attempt to call `xkb_keymap_new_from_file` on a null pointer.